### ISSUE TITLE
Attempting to create an MR on Gitlab throws an error

### ISSUE THREAD

**Issue by michaelaflores** [JCID/sourcetree-custom-action-gitlab-merge-request-opener#4]
I followed the setup instructions in the README. When running the shortcut command, I receive an error: 
```
sourcetree-merge-request-opener /Users/michaelflores/Documents/workspace/sam 
Couldn't posix_spawn: error 2
Completed with errors, see above
```

My settings are:
![screen shot 2018-01-18 at 1 44 21 pm](https://user-images.githubusercontent.com/11711706/35123116-d75ad21a-fc55-11e7-8c43-7a2b86c0798b.png)
The symlink to the script is here:
![screen shot 2018-01-18 at 1 45 45 pm](https://user-images.githubusercontent.com/11711706/35123139-f016c958-fc55-11e7-8d4b-8a2844b24919.png)

**Comment by cafferata** [comment]
Can you tell me what happens when you run the `sourcetree-merge-request-opener` binary on the CLI?
```
$ cd ~/Public/choose-an-git-repo/
$ php ~/.composer/vendor/bin/sourcetree-merge-request-opener .
```

**Comment by michaelaflores** [comment]
@cafferata when I run that, a window opens with a MR for Gitlab! Does this mean the issue is with Sourcetree itself or how I've added the script reference?

**Comment by mltk96** [comment]
same error here

**Comment by cafferata** [comment]
> Does this mean the issue is with Sourcetree itself or how I've added the script reference?

Unfortunately I can't reproduce the problem, which makes it extra difficult. After some searching on the Confluence of SourceTree I came across the following support question [On OSX Mavericks SourceTree errors](https://community.atlassian.com/t5/Sourcetree-questions/On-OSX-Mavericks-SourceTree-errors/qaq-p/350999). So can you try to [Wipe SourceTree Preferences](https://confluence.atlassian.com/sourcetreekb/how-to-wipe-sourcetree-preferences-412484640.html)?

**Please note that wiping the SourceTree preferences will mean you will lose your SourceTree bookmarks and view settings. Please back these up in the event that wiping your preferences isn't the cause of the problem.**

The following files need to be deleted after they've been backed up.

1. Make sure SourceTree is closed.
1. Make sure the files are backed up in the following folders before removing the files
1. Remove everything in `~/Library/Application Support/SourceTree/`
1. **If you're using an OS X version _before_ Mavericks**: Remove `~/Library/Preferences/com.torusknot.SourceTreeNotMAS.plist` (you should be using the direct version of SourceTree, hence "NotMAS")
1. If you're using Mavericks or later, run the following in the terminal **defaults delete com.torusknot.SourceTreeNotMAS**
1. Open SourceTree again to test if it worked or not.

If this didn't fix your problems then close SourceTree and restore the files, then re-open SourceTree again (if at all possible).

**Comment by stepharr** [comment]
@cafferata I'm having this issue with a git repo that has a submodule in it.

```sh
$ php ~/.composer/vendor/bin/sourcetree-merge-request-opener .

Warning: parse_ini_file(./.git/config): failed to open stream: No such file or directory in /Users/stephonharris/.composer/vendor/jcid/sourcetree-merge-request-opener/sourcetree-merge-request-opener on line 15
The file /Users/stephonharris/Projects/dir/repo/merge_requests/new?merge_request[source_branch]=-Fix-bugs does not exist.
```

**Comment by cafferata** [comment]
@stepharr thanks for taking time to report your problem! I have been able to reproduce the problem and solved it in release 1.0.3. Happy Easter!
- https://github.com/JCID/sourcetree-custom-action-gitlab-merge-request-opener/releases/tag/1.0.3

**Comment by cafferata** [comment]
@michaelaflores @mltk96 do you still experience problems? if so, can you help me with a reproducible environment?
